pkg/label/label.go

```diff
@@ -453,6 +447,7 @@ func (l Label) IsManagedByTiDBOperator() bool {
 	return l[ManagedByLabelKey] == TiDBOperator
 }
 
+// IsTidbClusterPod returns whether it's a TidbCluster controlled pod
```

Comment:
Is this what you mean?
```suggestion
// IsTidbClusterPod returns whether it is a TidbCluster-controlled Pod
```